packages/smooth_app/lib/pages/preferences_v2/tiles/url_preference_tile.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_app/helpers/launch_url_helper.dart';
+import 'package:smooth_app/pages/preferences_v2/tiles/preference_tile.dart';
+
+class UrlPreferenceTile extends PreferenceTile {
+  const UrlPreferenceTile({
+    required super.icon,
+    required super.title,
+    super.subtitleText,
+    required this.url,
+  });
+
+  final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    return PreferenceTile(
+      icon: icon,
+      title: title,
+      subtitleText: subtitleText,
+      trailing: const Icon(Icons.open_in_browser),
+      onTap: () {
+        LaunchUrlHelper.launchURLInWebViewOrBrowser(
+          context,
+          url,
+        );
+      },
```

Comment:
```suggestion
      onTap: () async =>
        LaunchUrlHelper.launchURLInWebViewOrBrowser(
          context,
          url,
        ),
```
Not sure what it would look like not awaiting for a WebView.